google/resource_compute_network.go

```diff
@@ -143,6 +190,55 @@ func resourceComputeNetworkRead(d *schema.ResourceData, meta interface{}) error
 	return nil
 }
 
+func resourceComputeNetworkUpdate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+
+	project, err := getProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	network := d.Get("name").(string)
+
+	d.Partial(true)
+
+	if d.HasChange("peering") {
```

Comment:
Right now, since I am changing the name of the peering, it does one add and one remove. I will add more coverage though. One problem is that I can't add more than 3 networks because by default, GCD account have a default network and a cap of 5 on the maximum number of networks. I like to leave at least one open to be able to allow some level of manual testing while my test run.